float32 mot1
float32 mot2
float32 mot3
